web/packages/teleport/src/Locks/useLocks.test.tsx

```diff
@@ -47,7 +47,7 @@ describe('hook: useLocks', () => {
     // happen before continuing on.
     await waitForNextUpdate();
     const resp = await result.current.createLock('cluster-id', {
-      targets: { user: 'banned' },
+      target: { user: 'banned' },
```

Comment:
Technically, a lock target can include multiple resources, hence the plural here.